src/borg/helpers/parseformat.py

```diff
@@ -662,7 +664,13 @@ def archive(self):
 
     def get_meta(self, key, rs):
         value = self.archive.metadata.get(key, '')
-        return remove_surrogates(value) if rs else value
+        if rs:
+            if isinstance(value, list):
+                return list(map(remove_surrogates, value))
```

Comment:
doesn't the code expect a str rather than a list?

Reply:
I think `json.dumps` should get the list while we want a neat string without JSON. It's implemented now.